App/populatePlaylist.js

```diff
@@ -0,0 +1,77 @@
+var playlistFromParse = JSON.parse(playlistsJSON);
+
+
+function unhidePlaylist(){
+
+    document.getElementById("playlist").style.visibility = "visible";
+    console.log(currentPlaylist);
+
+    var iDiv = document.createElement('div');
+    iDiv.id = 'content';
+    document.getElementById('content').appendChild(iDiv);
+
+
+//create header
+  for(var i = 0; i < currentPlaylist.songs.length; i++){
```

Comment:
use a forEach but only when map, reduce, filter is not more appropriate or when cannot be used. Anyhow, forEach  function is better than the for statement